webbpsf/utils.py

```diff
@@ -734,7 +735,7 @@ def to_griddedpsfmodel(HDUlist_or_filename=None, ext=0):
         ndd.meta['oversampling'] = ndd.meta['OVERSAMP'][0]  # pull the value
 
     # Remove keys with duplicate information
-    ndd.meta = {key.lower(): ndd.meta[key] for key in ndd.meta if 'DET_YX' not in key and 'OVERSAMP' not in key}
+    ndd.meta = {key.lower(): ndd.meta[key] for key in ndd.meta}
```

Comment:
With this change, the comment line immediately above about removing keys with duplicate information no longer applies. Instead this is just "turn all metadata keys into lowercase" now, I think.